src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -76,7 +77,7 @@ void BaseCouplingScheme::sendData(m2n::PtrM2N m2n, DataMap sendData)
   for (const DataMap::value_type &pair : sendData) {
     int size = pair.second->values().size();
     if (size > 0) {
-      m2n->send(pair.second->values().data(), size, pair.second->mesh->getID(), pair.second->getDimensions());
+      m2n->send(pair.second->values().data(), size, pair.second->getMeshID(), pair.second->getDimensions());
```

Comment:
How did this end up here?

Reply:
Also related to https://github.com/BenjaminRodenberg/precice/commit/f5c60edb2589544934fc8881c90d3dc1a550ee9e.

Reply:
This is cleaned up now via 03585b1. Especially 173dbdf.